### PR TITLE
Garbage collection in JS

### PR DESCRIPTION
**Pull request by AkaashK**
program to explain some concepts of Garbage collection in JS

### CODE REVIEW

File: akash/filters.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    
+    rl.question('Enter no of elements:',(number)=>{
+        recursiveReadline(number);
+    });
+});
+
+//Recursive function to get the array inputs from the user to the notes property in the object
+function recursiveReadline(number){
+    rl.question('Enter elements:',(elements)=>{
+        for(let i=0;i<number;i++){
```
Comment (MurugalakshmiM) [inline akash/filters.js]: check is this loop necessary

Comment (AkaashK) [reply akash/filters.js]: if loop is not needed and resolved the code without using if loop

**Review by MurugalakshmiM — APPROVED** [review]

File: akash/totalPrice.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const readline = require('readline')
+
+const read = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+})
+
+let productList = []
+
+let retry = function(){
```
Comment (MurugalakshmiM) [inline akash/totalPrice.js]: use const instead of let.

Comment (AkaashK) [reply akash/totalPrice.js]: replaced const instead of let and pr raised

**Pull request merged**